### PR TITLE
adds automatic resolution in Arquillian Cube Standalone #255

### PR DESCRIPTION
**Pull request by lordofthejars**

### CODE REVIEW

File: docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/AutoStartParserFactory.java [hunk 1]
```diff
@@ -5,17 +5,25 @@
...
+                    return new CommaSeparatedAutoStartParser(expression, containersDefinition);
+                } else {
+                    return null;
+                }
             }
         }
     }
 
+    private static boolean isNone(String expression) {
+        return expression != null && "[none]".equals(expression.trim());
```
Comment (aslakknutsen) [inline docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/AutoStartParserFactory.java]: Add description of "none" to doc/README

File: docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerExtension.java [hunk 2]
```diff
@@ -32,6 +32,9 @@ public void register(ExtensionBuilder builder) {
         if(Validate.classExists("org.jboss.arquillian.container.test.spi.client.deployment.AuxiliaryArchiveAppender")) {
             builder.observer(DockerServerIPConfigurator.class);
             builder.observer(CubeDockerAutoStartConfigurator.class);
+        } else {
+            // Arquillian in Standalone mode
+            builder.observer(StandaloneAutoStartConfigurator.class);
```
Comment (aslakknutsen) [inline docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerExtension.java]: Add the concept of autostart .* in standalone mode and why to doc/README

File: docker/ftest-standalone-ping/pom.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>arquillian-cube-docker-parent</artifactId>
+        <groupId>org.arquillian.cube</groupId>
+        <version>1.0.0.Final-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Arquillian Cube Docker Functional Test Standalone With PingPong</name>
+    <artifactId>arquillian-cube-docker-ftest-standalone-ping</artifactId>
```
Comment (aslakknutsen) [inline docker/ftest-standalone-ping/pom.xml]: Rename module to a name that describes what it is testing; e.g. standalone-autostart

File: docker/ftest-standalone-ping/src/test/resources/arquillian.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    xsi:schemaLocation="http://jboss.org/schema/arquillian
+    http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+    <extension qualifier="docker">
+        <property name="machineName">dev</property>
+        <property name="dockerContainers">
+            pingpong:
+              image: jonmorehouse/ping-pong
+              exposedPorts: [8080/tcp]
+              await:
```
Comment (aslakknutsen) [inline docker/ftest-standalone-ping/src/test/resources/arquillian.xml]: Default SS await doesn't work?

Comment (lordofthejars) [inline docker/ftest-standalone-ping/src/test/resources/arquillian.xml]: yes, it is a copy paste from `ftest-standalone`. I can push again fixing this, but basically I followed the standalone example.

**Pull request merged**